tests/run-make/allow-warnings-cmdline-stability/rmake.rs

```diff
@@ -0,0 +1,12 @@
+// Test that `-Awarnings` suppresses warnings for unstable APIs.
+
+use run_make_support::{assert_not_contains, rustc};
+
+fn main() {
+    rustc().input("bar.rs").run();
+    let output = rustc().input("foo.rs").arg("-Awarnings").command_output();
+
+    assert!(output.status.success());
```

Comment:
Suggestion: I think this can be `run()` instead of `command_output()`, because `run()` provided by `impl_common_helpers!(Rustc)` is

```rs
/// Run the constructed command and assert that it is successfully run.
#[track_caller]
pub fn run(&mut self) -> ::std::process::Output {
    let caller_location = ::std::panic::Location::caller();
    let caller_line_number = caller_location.line();

    let output = self.command_output();
    if !output.status.success() {
        handle_failed_output(&self.cmd, output, caller_line_number);
    }
    output
}
```

Reply:
We need the `output` below to get `stdout` and `stderr`. ;)

Reply:
Oh wait, nevermind. Yes you're absolutely right, good catch!